x-pack/plugins/security/server/config.ts

```diff
@@ -297,6 +297,11 @@ export const ConfigSchema = schema.object({
     ),
   }),
   enabled: schema.boolean({ defaultValue: true }),
+  ui: schema.object({
```

Comment:
Core recently introduced a change that only allows these to be configured within a serverless context. Can we update this to take advantage of that? More information for this contextRef: https://github.com/elastic/kibana/pull/156837

Reply:
Thanks, that change makes sense to me.

> Given that we will only configure ui when in serverless context, do you think we should give it a more descriptive name? Maybe uiServerless?

I'm OK to leave it as-is for the time being. I don't have a strong opinion either way on this one.